include/VecRng/MRG.h

```diff
@@ -0,0 +1,82 @@
+#ifndef VECRNG_MRG_H
+#define VECRNG_MRG_H 1
+
+/**
+  MRG: NameSpace for the MRG32k3a class based on RngStream.h(cpp)
+
+  RngStream is a class generating multiple streams of random numbers created
+  by Prof. Pierre L'Ecuyer, University of Montreal (lecuyer@iro.umontreal.ca) 
+  Original source codes of RngStream.h(cpp) is available at
+  http://www.iro.umontreal.ca/~lecuyer/myftp/streams00/c++/
+
+  Relevant articles in which MRG32k3a and the package with multiple streams 
+  were proposed:
+
+  P. L'Ecuyer, ``Good Parameter Sets for Combined Multiple Recursive Random
+  Number Generators'', Operations Research, 47, 1 (1999), 159--164.
+
+  P. L'Ecuyer, R. Simard, E. J. Chen, and W. D. Kelton, ``An Objected-Oriented
+  Random-Number Package with Many Long Streams and Substreams'', Operations
+  Research, 50, 6 (2002), 1073--1075
+  */
+
+#include "RngDefs.h"
+
+namespace vecRng {
+inline namespace VECRNG_IMPL_NAMESPACE {
+
+namespace MRG {
+
+  VECRNG_GLOBAL auto ndim    =   3;
+  VECRNG_GLOBAL auto vsize   =   6;
```

Comment:
Here it would be better to use an integer type explicitly, like `size_t`, for example.

Reply:
O.K.